examples/with-i18next/pages/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import { I18nextProvider } from 'react-i18next'
+import startI18n from '../tools/startI18n'
+import { getTranslation } from '../tools/translationHelpers'
+import Title from '../components/Title'
+
+export default class Homepage extends Component {
+  static async getInitialProps ({ req }) {
+    const isServer = !!req
+    const translations = await getTranslation('pt', 'common', 'http://localhost:3000/static/locales/')
```

Comment:
for the server-side the polyfill requires it to be an absolute url. I couldn't figure a way of going around that. That's why url is a parameter. I'm fully open to suggestions though...